pkg/authenticator/config/config.go

```diff
@@ -12,25 +12,27 @@ import (
 // Config defines the configuration parameters
 // for the authentication requests
 type Config struct {
-	Account             string
-	ClientCertPath      string
-	ContainerMode       string
-	ConjurVersion       string
-	PodName             string
-	PodNamespace        string
-	SSLCertificate      []byte
-	TokenFilePath       string
-	TokenRefreshTimeout time.Duration
-	URL                 string
-	Username            *Username
+	Account                     string
+	ClientCertPath              string
+	ContainerMode               string
+	SupportMultipleApplications string
```

Comment:
the rest of the params are alphabetical, can we maintain that?

Reply:
@sigalsax i don't really understand the purpose of this new param. how will it be used? can we maybe leverage the existing param `CONTAINER_MODE` for this purpose?

for example, let the customer set `CONTAINER_MODE=job` and query for that (i know that `job` is not a container mode).

Reply:
if that's the issue then we can add a param `DEPLOYMENT_MODE` and use that instead of `SUPPORT_MULTIPLE_APPLICATIONS` to be more generic.

Reply:
`DEPLOYMENT_MODE=job` is not sustainable or scalable because what happens when there's rotation, we are no longer a job. We are a deployment. It just gets a bit more confusing because what happens then if we run as a deployment but have another feature `DEPLOYMENT_MODE_deployment` will no longer be specific enough

We should start support flows instead of deployment types

Reply:
When we support rotation (later milestone), we will add
```
SUPPORT_MULTIPLE_APPLICATIONS=true
ROTATION=true
```
That way the flows are clear

Reply:
you still didn't answer my question - how will `SUPPORT_MULTIPLE_APPLICATIONS` be used? what will we do if it's `true`? what will we do if it's `false`?

I'm trying to understand your intention to give a better review.

Reply:
`SUPPORT_MULTIPLE_APPLICATIONS=true` means we will run as a job, if false we won't run as a Job and we expect to see `CONTAINER_MODE=init`
Because we are deploying with Helm, at this stage this env is not configurable. It will strictly be
```
- name: SUPPORT_MULTIPLE_APPS
   value: true
```
Usually for env var, we accept user input to fill in env var based on that input. For this env var, users will not be able to manipulate it. 
See this PR: https://github.com/cyberark/secrets-provider-for-k8s/pull/179